pkgs/applications/science/electronics/oregano/default.nix

```diff
@@ -0,0 +1,69 @@
+{ fetchFromGitHub
+, glib
+, goocanvas2
+, gnucap
+, groff
+, gtk3
+, gtksourceview
+, intltool
+, libxml2
+, lib
+, makeWrapper
+, ngspice
+, perl
+, pkg-config
+, python3
+, stdenv
+, wafHook
+, wrapGAppsHook
+, useNgspice ? false
+}:
+let
+  version = "0.84.43";
+in
+stdenv.mkDerivation {
+  pname = "oregano";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "drahnr";
+    repo = "oregano";
+    rev = "v${version}";
+    hash = "sha256-1GsL0N3O0clqdgkXoPKMhvW+y4Rzg4QSeOA54nH4kz4=";
+  };
+
+  nativeBuildInputs = [ makeWrapper wafHook wrapGAppsHook ];
+  patches = [ ./check-cfg-gio-unix.patch ];
+
+  buildInputs = [
+    glib
+    goocanvas2
+    groff
+    gtk3
+    gtksourceview
+    intltool
+    libxml2
+    perl
+    pkg-config
+    python3
+  ];
```

Comment:
`groff`, `perl`, `intltool`, `pkg-config` and `python3` should go into `nativeBuildInputs` as they run on the build platform when building (i.e. they need to be "native" packages when cross-compiling).

Reply:
Done